app/views/RoomView/index.tsx

```
@@ -1493,7 +1495,7 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 
 	render() {
 		console.count(`${this.constructor.name}.render calls`);
-		const { room, action, selectedMessages, isAutocompleteVisible } = this.state;
+		const { room, loading, action, selectedMessages, isAutocompleteVisible } = this.state;
```

ESLint

[error] 'loading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)